docs/source/_static/custom.css

```diff
@@ -19,7 +19,11 @@
 .wy-nav-side .wy-menu-vertical a:hover {
   color: #fff !important;
 }
-
+/* Sidebar item background color on hover */
+.wy-nav-side .wy-menu-vertical li.toctree-l1.current>a:hover,
+.wy-nav-side .wy-menu-vertical li.toctree-l1>a:hover {
+    background-color: #333333 !important; /* Dark gray background */
```

Comment:
nit - really quick, just noticed that the indent spacing here seems to be larger than in the others in the file

Reply:
Done ✅